worker/uniter/uniter_test.go

```diff
@@ -12,10 +12,12 @@ import (
 	"syscall"
 
 	corecharm "github.com/juju/charm/v8"
+	"github.com/juju/charm/v8/hooks"
 	"github.com/juju/errors"
 	"github.com/juju/loggo"
 	jc "github.com/juju/testing/checkers"
 	gc "gopkg.in/check.v1"
+	"gopkg.in/yaml.v3"
```

Comment:
We use yaml.v2 almost everywhere else I think. There appears to be a few places where v3 has crept in. Would be nice to be consistent.

Reply:
Weren't there some reasons to not move to yaml v3?  But we needed it in a very specific case.

Reply:
yeah thanks for that, this was just an autoimport issue